### DISCUSSION TITLE
state of/plans for ssh to remote, and how do people manage it?

### DISCUSSION THREAD

**Discussion by blex-max** [helix-editor/helix#10875] [Q&A]
I know this isn't a novel question but I'm interested in the latest answer/thoughts - I work with helix in an environment where I ssh to a lot of different remotes. vscode really shines here, because you can connect to the remote and it will handle LSPs for you more or less in the background, installing what is needed on a per project basis on the remote in the `.vscode` folder (I think, I've never actually dug into this to be honest). How do people handle this situation with helix? My current solution is that I have a container with helix and all my LSPs accessible from a location available to all my remotes, with the option to add to path when running the container to point helix at e.g. python virtual envs. What would really set helix apart for me would be if it had good support for ssh-ing like this. Thanks very much!

**Comment by kirawi** [comment]
See https://github.com/helix-editor/helix/discussions/10411

I intend to eventually implement it as a plugin using distant at some point, but likely not this year at the very least.
